delta-lake/common/src/main/databricks/scala/com/nvidia/spark/rapids/delta/RapidsDeltaUtils.scala

```diff
@@ -53,6 +55,42 @@ object RapidsDeltaUtils {
       }
     }
 
+    // Optimized writes for non-partitioned tables involves a round-robin partitioning, and that
+    // can involve a sort on all columns. The GPU doesn't currently support sorting on all types,
+    // so we fallback if the GPU cannot support the round-robin partitioning.
+    if (sqlConf.sortBeforeRepartition) {
+      val orderableTypeSig = (GpuOverrides.pluginSupportedOrderableSig + TypeSig.DECIMAL_128
```

Comment:
Do we want to make a val that combines all of these and is used here and in GpuOverides SortOrder so that if we change in GpuOverrides we make this picks it up?

Reply:
Yes, very much so, but I think that's outside the scope of this PR.  I filed #7807 to fix that along with a bug in ShuffleExchangeExecBase related to the fact that there isn't a proper val for this.